benchmarking/benchmark.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
```

Comment:
This is really nice. Also getting somewhat complex; should we look into using an actual test framework? We have this AI think to generate tests that we could give a shot? Or even just:

https://developer.android.com/training/testing/other-components/ui-automator